src/test/resources/allfields/input/instructions.csv

```diff
@@ -1,11 +1,11 @@
-ruimtereis02,Hubble ruimtetelescoop,,Dr,C,,Sagan,,Cornell University,RightsHolder,,,,,,,,2015-05-13,paul,,"De Hubble-ruimtetelescoop (Hubble Space Telescope, HST) bestaat uit een aantal precisie-instrumenten voor astronomische waarnemingen. Hij is genoemd naar de Amerikaanse astronoom Edwin Hubble, en draait sinds de lancering door de NASA op 24 april 1990 als een kunstmaan rond de aarde.",astronomie,,2015-2016,,,,,,,,,,ds2,,isVersionOf,context,barbapappa,2017-08-01,Date Accepted,,image/jpeg,day,http://scholar.google.com,OPEN_ACCESS,,D30000,path/to/images/Hubble_01.jpg,Hubble,RESTRICTED_REQUEST,,,,,,,,user001
-ruimtereis02,Hubble space telescope,,,A.W.M.,van,tseeH,info:eu-repo/dai/nl/012345678,WANK/SNAD,ProjectManager,,,,,,,,2015-05-13,henk,,"The Hubble Space Telescope (HST) is a space telescope that was launched into low Earth orbit in 1990, and remains in operation. With a 2.4-meter (7.9 ft) mirror, Hubble's four main instruments observe in the near ultraviolet, visible, and near infrared spectra. The telescope is named after the astronomer Edwin Hubble.",telescopen,,,,,,,,,,,,0925-6229,ISSN,,,,random text,,,text/plain,dut,,OPEN_ACCESS,,,,,,,,,,,,,user001
-ruimtereis02,,,,,,,,,,,,,,,,,2015-05-13,richard,,,,,,,,,,,,,,,,,,,,,,,,,,OPEN_ACCESS,,,,,,,,,,,,,user001
-ruimtereis02,,,,,,,,,,,,,,,,,2015-05-13,joke,,,,,,,,,,,,,,,,,,,,2017-08-02,Date Copyrighted,,,,,OPEN_ACCESS,,,,,,,,,,,,,
-ruimtereis02,,,,,,,,,,,,,,,,,2015-05-13,vesa,,,,,,,,,,,,,,,,,,,,,,,,,,OPEN_ACCESS,,,,,,,,,,,,,
-ruimtereis03,Title for empty dataset,,,Donald,,Duck,,Duckstad,,,,,,,,,2016-07-30,unknown person,,This is an entry for a dataset with no data in it,empty space,,,,,,,,,,,,,,,,,,,,,,,GROUP_ACCESS,,D37000,,,,,,,,,,,user001
-ruimtereis04,Audio Dataset,,,Guus,,Geluk,,Duckstad,,,,,,,,,2016-07-31,,,This is a dataset with audio in it,,,,,,,,,,,,,,,,,,,,,audio/mpeg3,,,OPEN_ACCESS_FOR_REGISTERED_USERS,2016-07-30,D30000,path/to/a/random/sound/chicken.mp3,our daily wake up call,,dans,janvanmansum,Jans-test-files,continuous,,,,user001
+DATASET,DC_TITLE,DCT_ALTERNATIVE,DCX_CREATOR_TITLES,DCX_CREATOR_INITIALS,DCX_CREATOR_INSERTIONS,DCX_CREATOR_SURNAME,DCX_CREATOR_DAI,DCX_CREATOR_ORGANIZATION,DCX_CREATOR_ROLE,DCX_CONTRIBUTOR_TITLES,DCX_CONTRIBUTOR_INITIALS,DCX_CONTRIBUTOR_INSERTIONS,DCX_CONTRIBUTOR_SURNAME,DCX_CONTRIBUTOR_DAI,DCX_CONTRIBUTOR_ORGANIZATION,DCX_CONTRIBUTOR_ROLE,DDM_CREATED,DCT_RIGHTSHOLDER,DC_PUBLISHER,DC_DESCRIPTION,DC_SUBJECT,DC_SUBJECT_SCHEME,DCT_TEMPORAL,DCT_TEMPORAL_SCHEME,DCT_SPATIAL,DCX_SPATIAL_SCHEME,DCX_SPATIAL_X,DCX_SPATIAL_Y,DCX_SPATIAL_NORTH,DCX_SPATIAL_SOUTH,DCX_SPATIAL_EAST,DCX_SPATIAL_WEST,DC_IDENTIFIER,DC_IDENTIFIER_TYPE,DCX_RELATION_QUALIFIER,DCX_RELATION_TITLE,DCX_RELATION_LINK,DCT_DATE,DCT_DATE_QUALIFIER,DC_TYPE,DC_FORMAT,DC_LANGUAGE,DC_SOURCE,DDM_ACCESSRIGHTS,DDM_AVAILABLE,DDM_AUDIENCE,FILE_PATH,FILE_TITLE,FILE_ACCESSIBILITY,SF_DOMAIN,SF_USER,SF_COLLECTION,SF_PLAY_MODE,AV_FILE_PATH,AV_SUBTITLES,AV_SUBTITLES_LANGUAGE,DEPOSITOR_ID,BASE_REVISION
```

Comment:
1. let's only put base revisions in `deposit1` and `deposit4`; then we also have test cases without a base revision (which is also still valid, of course)
2. in `deposit1` use the same base revision in multiple rows
2a. change the output's (`bag-info.txt`) accordingly
3. change the `tagmanifest-sha1.txt` files of deposit1 and deposit4 accordingly
4. sync the CSV with the Excel sheet